diff --git a/dist/types-CFG827XN.mjs b/dist/types-CFG827XN.mjs
index cd219a5c64a0926d4474909ca0abeabc609bc6ed..7f139c0fe16eeae1c2391c24318b87ac44897a2b 100644
--- a/dist/types-CFG827XN.mjs
+++ b/dist/types-CFG827XN.mjs
@@ -3050,7 +3050,7 @@ async function getVersions(spec, options = {}) {
 	return getVersionsForContext(context, options);
 }
 function getVersionsForContext(context, options) {
-	if (isDefaultRegistry(context.registry)) return getVersions$1(context.parsed.raw, toDefaultOptions(options));
+
 	return getVersionsFromRegistry(context, options);
 }
 function toDefaultOptions(options) {
